internal/pkg/cli/pipeline_deploy.go

```diff
@@ -95,19 +106,62 @@ func newDeployPipelineOpts(vars deployPipelineVars) (*deployPipelineOpts, error)
 		prog:               termprogress.NewSpinner(log.DiagnosticWriter),
 		prompt:             prompt.New(),
 		codestar:           cs.New(defaultSession),
+		newSvcListCmd: func(w io.Writer) actionCommand {
```

Comment:
Instead of `actionCommand`, what about [`cmd`](https://github.com/aws/copilot-cli/blob/d321374c655428b3665d46f5f562ea08caff5b0c/internal/pkg/cli/interfaces.go#L41)?